Modules/_cursesmodule.c

```diff
@@ -159,16 +159,26 @@ typedef chtype attr_t;           /* No attr_t type is available */
 #define _CURSES_PAIR_CONTENT_FUNC       pair_content
 #endif  /* _NCURSES_EXTENDED_COLOR_FUNCS */
 
+typedef struct _cursesmodule_state {
+    PyObject *PyCursesError;
+    PyTypeObject *PyCursesWindow_Type;
```

Comment:
Maybe rename them to "error" and  "window_type", what do you think?

Reply:
Mmh, good idea!